### ISSUE TITLE
I cant open any server nor stop the process

### ISSUE THREAD

**Issue by ghost** [team9000/misc#531]
bora reported:

I tried to open a Gmod server . I logged in and pressed "Start".It said 
Machine mantis:0000001 - garrysmod - STARTING . After 1 hour, It still didnt work so I tried to shut it down but now it says
"Shutting down... [NOT YET STARTED]
Destroying... [NOT YET STARTED]
Shutting down... [NOT YET STARTED]
Destroying... [NOT YET STARTED]
Shutting down... [NOT YET STARTED]
Destroying... [NOT YET STARTED]
Shutting down... [NOT YET STARTED]
Destroying... [NOT YET STARTED]
Shutting down... [NOT YET STARTED]
Destroying... [NOT YET STARTED] "

So basically Im stuck and cant get out.

Mantis project: **general**
Mantis ID: **653**
